packages/components/src/presentation-components/properties/UseNavigationPropertyTargetsLoader.ts

```diff
@@ -22,7 +22,7 @@ import {
 import { Presentation } from "@itwin/presentation-frontend";
 
 /** @internal */
-export const NAVIGATION_PROPERTY_TARGETS_BATCH_SIZE = 100;
+export const PRESENTATION_TARGETS_BATCH_SIZE = 100;
```

Comment:
I would suggest having separate `BATCH_SIZE` constants for navigation property and unique values. Otherwise I would move this to some common place to avoid UniqueValuesSelector having to import from NavigationPropertySelector.

Reply:
fixed